applications/luci-app-attendedsysupgrade/luasrc/view/attendedsysupgrade.htm

```diff
@@ -181,7 +180,7 @@ <h2 name="content"><%:Attended Sysupgrade%></h2>
 // initial setup, get system information
 function setup() {
 	data["ubus_rpc_session"] = "<%=luci.dispatcher.context.authsession%>"
-	ubus_call("packagelist", "list", {}, "packagelist");
+	ubus_call("rpc-sys", "pacagelist", {}, "packagelist");
```

Comment:
I don't think you actually tested this ("pacagelist" != "packagelist")

Reply:
Fair enough! I think I accidentally hit `u` in vim or something. Very strange. It used to work, I swear :innocent: